hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMultiTenantManagerImpl.java

```diff
@@ -332,7 +335,7 @@ public String assignUserToTenant(String tenantName, String userName) {
       inMemoryUserNameToListOfGroupsMap.put(
           userPrincipal.getFullMultiTenantPrincipalID(), userGroupIDs);
 
-      return userID;
+      return null;
```

Comment:
Is this intentional, or for testing? Maybe put a TODO here?

Reply:
DEBUG change, removed.